packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1710,6 +1708,7 @@ describe("Runtime", () => {
 						// Submit an op and yield for it to be flushed from outbox to pending state manager.
 						submitDataStoreOp(containerRuntime, "fakeId", "fakeContents");
 						await yieldEventLoop();
+						// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
Unfortunately, `no-unsafe-argument` doesn't have an escape hatch for rest args. Still likely worth keeping as a rule, though, as these scenarios should generally not be _too_ common.

Reply:
Can you clarify what you mean by this?

Reply:
Check out [this thread in a previous PR](apiItem.parent?.kind === ApiItemKind.Package) for more context.

We allow `any` to be used for rest argument parameters for `@typescript-eslint/no-explicit-any`. But this rule doesn't offer a similar exception, unfortunately. So while the place where `args` is defined is allowed by the linter, it complains when we pass it into another function.